### PR TITLE
Logging returns `true`

### PR DESCRIPTION
**Pull request by fnordfish**
See #64 

## Types of Changes

- Logging methods (like `Console.info`) consistently return `true` regardless of Output
 
## Contribution

- [x] I added tests for my changes.
- [x] I tested my changes locally.
- [x] I agree to the [Developer's Certificate of Origin 1.1](https://developercertificate.org/).

### CODE REVIEW

**Comment by fnordfish** [comment]
Just realized, that this is not complete yet. A log not matching the current filter will still return `nil`. For example, with log level info `Console.debug("test")` will return `nil`

**Comment by ioquatix** [comment]
This looks somewhat reasonable to me.

I think if we had to establish a behaviour, it should be that the log operation returns truthy if it does something and falsey otherwise.

I believe `if filter ... end` already returns falsey if it isn't logged.

Regarding the other cases, I'm okay for them to return true if something was logged or not.

What do you think?

**Comment by fnordfish** [comment]
I have two thought:

1st: I assume `::Logger` uses `true` as a universal return value to allow for `log("message") and do_something_else` 

2nd: Personally, I like to have a consistent return value. My code should not need to care if the log has been written or not. 
Consider this code
```ruby
def do_something
  # more code
  Console.error("we did something") unless it_was_done
end
```
Now, in tests and development, you'll probably have error logging on, but in production, you might raise the log lever to fatal. Suddenly your method would return a different value.  
If we ensure logging returns a uniform value, application code can rely on that and do what's right.

I've went ahead with returning `true` just because Console already returned truthy values (and a little bit to mimic `::Logger`). I would be totally fine changing this and returning `nil` everywhere.

File: test/console/filter.rb [hunk 1]
```diff
@@ -70,25 +74,28 @@ def before
 		it "ignores messages below the level" do
 			logger.level = Console::Logger::INFO
 			
-			logger.call(MySubject, "Hello World", severity: :debug)
+			retval = logger.call(MySubject, "Hello World", severity: :debug)
```
Comment (ioquatix) [inline test/console/filter.rb:77]:
```suggestion
			result = logger.call(MySubject, "Hello World", severity: :debug)
```

**Comment by ioquatix** [comment]
If we are planning to return nothing why not always return nil?

File: test/console/filter.rb (hunk 1 above)
Comment (fnordfish) [reply test/console/filter.rb]: fixed

**Comment by ioquatix** [comment]
I've decided to go ahead with always returning `nil` from `Console::Filter`.

Honestly, I could see different ways this could be implemented. Returning `true`/`false` is also interesting, because it might allow conditional behavior (if this was not logged, do something else). The question of whether something is logged or not, however, IMHO, should not affect program behaviour. So, I decided not to encourage that. In addition, returning true might encourage users to use `Console...` in a conditional and I also don't want to encourage that either.

**Comment by ioquatix** [comment]
Thanks for your contribution, closing in favour of #68.

**Pull request closed without merging**